debian/kolibri-server.service

```diff
@@ -10,12 +10,7 @@ Wants=network-online.target
 Type=forking
 ExecStart=/etc/init.d/kolibri-server start
 ExecStop=/etc/init.d/kolibri-server stop
-TimeoutStartSec=infinity
```

Comment:
Was it also the intention to remove this?

Reply:
Yes, after removing the `service` call in the service script this was unneeded. Default system values are more than enough.